<launch>
  <!-- Launch the motion detector node for image processing. -->
  <node pkg="motion_detection" name="MotionDetection" type="motion_detector.py"/>
  <!-- Launch the driver node for our usb camera. -->
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video1" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
    <param name="framerate" value="30"/>
  </node>
  <node name="rviz" pkg="rviz" type="rviz" required="true" args="-d $(find motion_detection)/rviz/motion.rviz" />
</launch>